src/main/java/com/mosaicchurchaustin/oms/services/OrderService.java

```diff
@@ -281,4 +286,42 @@ private void addItemsToOrder(final OrderEntity orderEntity, List<OrderItemReques
         }
     }
 
+    public String tryGetSerializedOrder(OrderEntity order) {
+        try {
+            return getSerializedOrder(order);
+        } catch (Exception e) {
+            return "";
+        }
+    }
+
+    private String getSerializedOrder(OrderEntity order) {
+        StringBuilder sb = new StringBuilder();
+
+        // Add customer name
+        sb.append("Friend: ").append(order.getCustomer().getName()).append("\n\n");
+
+        // Add order items
+        sb.append("Order Items:\n");
+        for (OrderItemEntity orderItem : order.getOrderItemList()) {
+            sb.append("- ")
+              .append("x").append(orderItem.getQuantity())
+              .append(" ")
+              .append(orderItem.getItemEntity().getDescription());
+
+            // Include item-specific notes if present
+            if (orderItem.getNotes() != null && !orderItem.getNotes().isEmpty()) {
+                sb.append(" [").append(orderItem.getNotes()).append("]");
+            }
+
+            sb.append("\n");
+        }
+
+        // Add special instructions for the order if any
+        if (order.getSpecialInstructions() != null && !order.getSpecialInstructions().isEmpty()) {
```

Comment:
I'd make this a bit more readable with something like:
Optional.ofNullable(StringUtiels.defaultIfEmpty(order.getSpecialInstructions(), null).ifPresent(instructions -> {
sb.append("\nSpecial Instructions: ")
.append(order.getSpecialInstructions())
.append("\n")
})